src/m365/aad/commands.ts

```diff
@@ -5,6 +5,7 @@ export default {
   ADMINISTRATIVEUNIT_GET: `${prefix} administrativeunit get`,
   ADMINISTRATIVEUNIT_LIST: `${prefix} administrativeunit list`,
   ADMINISTRATIVEUNIT_REMOVE: `${prefix} administrativeunit remove`,
```

Comment:
Which command should be moved? `ADMINISTRATIVEUNIT_REMOVE`? It can be in conflict with [src/m365/aad/commands.ts](https://github.com/pnp/cli-microsoft365/pull/5652/files/d76cbdaf41f4c45ee449655bea4a77ada98fd21a#diff-29134e4ea847836a64858544411731026e03f3b3f1c8a132fa4d261dede363d0) where I moved `ADMINISTRATIVEUNIT_MEMBER_LIST` below `ADMINISTRATIVEUNIT_REMOVE`

Reply:
I listed the commands alphabetically

Reply:
Hi @MartinM85, It's not a big thing, but we try to list the commands first by commandgroup and only then alphabetically. Same as with the docs sidebar.

Reply:
In that case, the previous code was correct

```
ADMINISTRATIVEUNIT_ADD...
ADMINISTRATIVEUNIT_GET...
ADMINISTRATIVEUNIT_LIST...
ADMINISTRATIVEUNIT_REMOVE...
ADMINISTRATIVEUNIT_MEMBER_ADD...
ADMINISTRATIVEUNIT_MEMBER_LIST...
```

Changed

Reply:
ah, I think I was looking at the wrong entry... in that case never mind :-)